raft/src/test/java/org/apache/kafka/raft/KafkaRaftClientSnapshotTest.java

```diff
@@ -48,6 +48,9 @@
 import static org.junit.jupiter.api.Assertions.assertTrue;
 
 final public class KafkaRaftClientSnapshotTest {
+
+    private static final int REPLICA_FETCH_RESPONSE_MAX_BYTES = 8 * 1024 * 1024;
```

Comment:
Why can't we use `KafkaRaftClient.MAX_BATCH_SIZE_BYTES` here?

Reply:
As far as I understand, KafkaRaftClient.MAX_BATCH_SIZE_BYTES is not used for fetchSnapshot response limit size, it is used in accumulator test.
Correct me if I misunderstood, thanks